src/harness/request_handler.py

```diff
@@ -75,15 +75,27 @@ def WithClientCertificate(self, client_cert, client_key):
     return ret
 
 
-def RequestPost(url, request, config):
-  return _Request(url, request, config, True)
+def RequestPostEmpty(url, config):
+  return _Request(url, None, config, True, None)
 
 
-def RequestGet(url, config):
-  return _Request(url, None, config, False)
+def RequestPostJson(url, request, config):
+  return _Request(url, request, config, True, 'application/json')
 
 
-def _Request(url, request, config, is_post_method):
+def RequestGetEmpty(url, config):
+  return _Request(url, None, config, False, None)
+
+
+def _RequestPost(url, request, config, content_type):
+  return _Request(url, request, config, True, content_type)
+
+
+def _RequestGet(url, request, config):
+  return _Request(url, request, config, False)
```

Comment:
_RequestPost and _RequestGet aren't used by anything, correct? Pretty sure the only functions the should be called directly for GET and POST should be:
- RequestPostEmpty (Not sure this is the best name, but naming is hard)
- RequestPostJson
- RequestGet (No need to change it to RequestGetEmpty)